packages/core/certora/optimistic-asserter/specs/Asserter_Bonds.spec

```diff
@@ -0,0 +1,186 @@
+// For every new spec you create, import this basic spec file:
+import "./Asserter_Base.spec"
+import "./dispatchedMethods.spec"
+
+// An ERC20 contract in our scope. One can access its methods or address by
+// its alias (testERC20)
+using TestnetERC20 as testERC20
+/**************************************************
+ *      Top Level Properties / Rule Ideas         *
+ **************************************************/
+ // Prove that the sum of bonds for a specific currency is correlated with the 
+ // main contract ERC20 balance. 
```

Comment:
This is an awesome property. +1

Reply:
I agree :)
It's similar to solvency in liquidity pools or some bank-like protocols we already tested in the past, so this was trivial for us to think about.
Since the bonds aren't updated back to zero once the truth is asserted, we might need to think about how to implement this rule exactly.